### ISSUE TITLE
`globalThis.GM_*` is `undefined` in Firefox

### ISSUE THREAD

**Issue by AlttiRi** [violentmonkey/violentmonkey#1463]
I have:
```js
// @grant       GM_registerMenuCommand
```
I use:
```js
console.log("globalThis.GM_registerMenuCommand", globalThis.GM_registerMenuCommand);
```

In Firefox `globalThis.GM_registerMenuCommand` is `undefined`. 

In Chromium it works OK.
In Firefox + TM it also works OK.

### Environment
- Browser: Firefox
- Browser version: 97
- Violentmonkey version: v2.13.0
- OS: Win 10

**Comment by tophf** [comment]
I guess you are running this code in `content` mode, which activates automatically on sites with strict CSP that forbids script injection. Tampermonkey disables CSP of such sites, we don't.

I'll fix it by explicitly mapping `globalThis` to our wrapper.

**Comment by AlttiRi** [comment]
No, with the default mode.

**Comment by tophf** [comment]
By default the `auto` mode is used, which works as I described above on sites with strict CSP, so your script was running in `content` mode.

**Comment by tophf** [comment]
Fixed in b8554b4f.
